co2calculator/api/energy.py

```diff
@@ -57,19 +62,19 @@ def from_electricity(self, country_code: str):
             country_code=country_code,
         )
 
-    def from_heating(self, unit: str):
+    def from_heating(self, in_kwh: bool = False):
         """Calculate emissions from heating consumption
 
-        :param unit: unit of measurement for heating consumption
-        :type unit: str
+        :param in_kwh: if True, consumption is in kWh
         """
-        if unit is not None and not isinstance(unit, str):
-            raise ValueError("unit must be a string")
+        if self.fuel_type is None and not in_kwh:
```

Comment:
Looking at it again, I feel like it would be more intuitive for the user if we pass all parameters in the `from_heating` function. So the `Energy` class wouldn't take any parameters and would only be a container for the methods.